app/models/user.rb

```diff
@@ -165,6 +165,10 @@ def can_moderate?
     admin? || moderator?
   end
 
+  def is_associated(node)
```

Comment:
Maybe this should be called `.is_coauthor` since "associated" has some other meanings?